jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/CloudWatchLogStream.kt

```diff
@@ -90,6 +94,25 @@ class CloudWatchLogStream(
         Disposer.register(this, logStreamActor)
         searchLabel.text = "${project.activeCredentialProvider().displayName} => ${project.activeRegion().displayName} => $logGroup => $logStream"
         searchField.emptyText.text = message("cloudwatch.logs.filter_logs")
+        searchField.addActionListener(object : ActionListener {
+            private var lastText: String? = null
+            override fun actionPerformed(e: ActionEvent?) {
+                if (searchField.text == lastText) {
+                    return
+                }
+                lastText = searchField.text
+                val client: CloudWatchLogsAsyncClient = project.awsClient()
+                launch {
+                    val response = client.filterLogEvents {
+                        it
+                            .logGroupName(logGroup)
+                            .logStreamNames(logStream)
+                            .filterPattern(lastText)
+                    }.await()
+                    val events = response.events()
```

Comment:
Where does this go? It seems to be dangling...

Reply:
This was committed to the wrong branch, removed